modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNode.tsx

```diff
@@ -53,7 +53,6 @@ export function ObjectDefinitionNode({
 		externalReferenceCode,
 		hasObjectDefinitionDeleteResourcePermission,
 		hasObjectDefinitionManagePermissionsResourcePermission,
-		hasSelfObjectRelationships,
```

Comment:
@rebsilva Apparently, this property only had the purpose of controlling the creation of those handles. Since it won't be needed anymore, please remove it from other files it was declared/initialized.

Reply:
Ohh I understand now, I'll do it!